sixMans/game.py

```diff
@@ -603,6 +622,9 @@ async def post_more_lobby_info(self, helper_role=None, invalid=False):
         # embed.description = player_scores_str
         self.info_message = await self.textChannel.send(embed=embed)
 
+    def has_lobby_info(self):
+        return self.roomName and self.roomPass and len(self.blue) > 0 and len(self.orange) > 0
```

Comment:
I think we should change this to
```py
return self.roomName and self.roomPass and len(self.blue) == self.queue.max_size and len(self.orange) == self.queue.max_size
```
You should double check the variable name, but you get the idea. We don't _really_ have lobby info until teams are full, with all players placed.

Reply:
You mean this right
```suggestion
        return self.roomName and self.roomPass and len(self.blue) + len(self.orange) == self.queue.max_size
```

Reply:
Yes, either that or
```py
len(self.blue) == self.queue.max_size and len(self.orange) == self.queue.max_size
```